proto/substrait/algebra.proto

```diff
@@ -225,6 +225,9 @@ message AggregateRel {
     // within the measure.
     // Helps to support SUM(<c>) FILTER(WHERE...) syntax without masking opportunities for optimization
     Expression filter = 2;
+    // An optional sort expression that acts to define required ordering for the aggregator.
+    // Helps to support ARRAY_AGG(<c> ORDER BY <C2>) syntax without masking opportunities for optimization
+    Expression order_by = 3;
```

Comment:
We don't use a plain `Expression` to define ordering elsewhere.  For example, see the `SortRel` which uses `repeated SortField`.  Is there a particular reason you are using `Expression` here?

Reply:
This makes sense to me, also it is similar to the pattern in `WindowFunction`. Also using `repeated` enables us to support lexicographical requirement. I have converted `Expression order_by` to `repeated SortField order_by`. Thanks for the suggestion.